app/assets/javascripts/pretty-text/addon/engines/discourse-markdown-it.js

```diff
@@ -578,14 +579,16 @@ export function setup(opts, siteSettings, state) {
 
 export function cook(raw, opts) {
   // we still have to hoist html_raw nodes so they bypass the allowlister
```

Comment:
The fact we bypass the allowlister seems risky to me. Maybe it is something safe, but it looks odd to me.

Reply:
I think it's fine, this was from a onebox fix here https://github.com/discourse/discourse/commit/df5ee0cfc1ff932f713d418eb9cb34dfb67a74cd and I don't see any reason to mess with it